app/controllers/recipes_controller.rb

```diff
@@ -1,28 +1,25 @@
 class RecipesController < ApplicationController
-
-
-before_filter :set_default_response_format
+  before_action :authenticate
 
   # GET /recipes
   # GET /recipes.json
   def index
-    @recipes = Recipe.all
-
-    render
+    @recipes = @current_user.recipes.load
+    render json: @recipes
   end
 
   # GET /recipes/1
   # GET /recipes/1.json
   def show
-    @recipe = Recipe.find(params[:id])
+    @recipe = @current_user.recipes.find(params[:id])
```

Comment:
I agree that there should be a reader method and that we should use that method, but I'm not sure if the whole reason for it is so that I can search for `def current_user` in my editor. :stuck_out_tongue: